porcelain/miner.go

```diff
@@ -330,7 +331,7 @@ func MinerGetSectorSize(ctx context.Context, plumbing minerQueryAndDeserialize,
 
 // MinerCalculateLateFee calculates the fee due if a miner's PoSt were to be mined at `height`.
 func MinerCalculateLateFee(ctx context.Context, plumbing minerQueryAndDeserialize, minerAddr address.Address, height *types.BlockHeight) (types.AttoFIL, error) {
-	abiVal, err := queryAndDeserialize(ctx, plumbing, minerAddr, "calculateLateFee", height)
+	abiVal, err := queryAndDeserialize(ctx, plumbing, minerAddr, "calculateLateFee", plumbing.ChainHeadKey(), height, plumbing.ChainHeadKey())
```

Comment:
In this and some of the other methods below (inspecting state that does change over time), the head key should be a parameter to the porcelain function. This can wait for a follow-up.